src/linter/rules/invalid-change-version.mjs

```diff
@@ -19,11 +19,11 @@ export const invalidChangeVersion = entry => {
     );
 
   const invalidVersions = allVersions.filter(
-    version => valid(version) === null
+    version => version !== 'REPLACEME' && valid(version) === null
```

Comment:
I would filter out `REPLACEME` before reaching here, and we need to support passing `NODE_RELEASED_VERSIONS` via the env
```suggestion
    process.env.NODE_RELEASED_VERSIONS ?
      version => process.env.NODE_RELEASED_VERSIONS.includes(version) :
      version => !valid(version),
```